src/index.js

```diff
@@ -199,10 +199,32 @@ function handleBody(value, config) {
     return ['div', {}, 'body'];
 }
 
-export function register(opts = { simple_mode: false, debug: false }) {
+function hasFormatterSupport() {
+    const originalFormatters = window.devtoolsFormatters;
+    let supported = false;
+
+    window.devtoolsFormatters = [
+        {
+            header: function(obj, config) {
+                supported = true;
+                return null;
+            },
+            hasBody: function(obj) {},
+            body: function(obj, config) {},
+        },
+    ];
+    console.log('elm-debug-transformer: checking for formatter support.', {});
```

Comment:
I'd probably drop this `console.log`. It is tricky to be hooked on the `console` output and write something to it as it would try to go through the parser as well and one can be stuck in the loop.

Reply:
Hmm on the other hand it is outside of the function that hooks the `console.log`, so it is probably OK

Reply:
I can't think of a way to do the feature detection without it. Calling it and seeing if the temporary formatter ran and set `supported` to true is what's doing all the work here. 

It's unfortunate that formatters aren't registered with a function call that we could inspect for, as that would be much easier.  I don't know anybody with Chrome merge rights though. :grin: .